portal/import_template_ui.php

```diff
@@ -422,27 +436,31 @@ function popGroupsDialog() {
-                <?php if ($authUploadTemplates) { ?>
-                    <form id='form_upload' class='form-inline row' action='import_template.php' method='post' enctype='multipart/form-data'>
-                        <hr />
-                        <div class='col'>
-                            <div id='upload_scope_category'></div>
-                            <div class='mb-2' id='upload_scope'></div>
-                        </div>
-                        <div class='form-group col'>
-                            <div class='form-group'>
-                                <input type="hidden" name="csrf_token_form" id="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken('import-template-upload')); ?>" />
-                                <input type='file' class='btn btn-outline-info' id="fetch_files" name='template_files[]' multiple />
-                                <button class='btn btn-outline-success d-none' type='submit' name='upload_submit' id='upload_submit'><i class='fa fa-upload' aria-hidden='true' mr-1></i><?php echo xlt("Templates"); ?></button>
-                                <button class='btn btn-outline-success d-none' type='submit' name='upload_submit_questionnaire' id='upload_submit_questionnaire'><i class='fa fa-upload aria-hidden='true' mr-1'></i><?php echo xlt("Questionnaires Only"); ?></button>
+                    <?php if ($authUploadTemplates) { ?>
+                        <form id='form_upload' class='form-inline row' action='import_template.php' method='post' enctype='multipart/form-data'>
+                            <hr />
+                            <div class='col'>
+                                <div id='upload_scope_category'></div>
+                                <div class='mb-2' id='upload_scope'></div>
                             </div>
-                        </div>
-                        <input type='hidden' name='upload_pid' value='<?php echo attr(json_encode([-1])); ?>' />
-                        <input type='hidden' name="template_category" value='<?php echo attr($category); ?>' />
-                    </form>
-                <?php } else { ?>
-                    <div class="alert alert-danger"><?php echo xlt("Not Authorized to Upload Templates") ?></div>
-                <?php } ?>
+                            <div class='form-group col'>
+                                <div class='form-group'>
+                                    <input type="hidden" name="csrf_token_form" id="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken('import-template-upload')); ?>" />
+                                    <input type='file' class='btn btn-outline-info' id="fetch_files" name='template_files[]' multiple />
+                                    <button class='btn btn-outline-success d-none' type='submit' name='upload_submit' id='upload_submit' title="<?php echo xlt("Import a template file or if a Questionnaire then auto create a questionnaire template."); ?>">
```

Comment:
xla here and in line 452 below
(this made my day :) , the first escaping advice i have been able to give in your PR's in a very long time)